.travis.yml

```diff
@@ -1,19 +1,22 @@
+dist: bionic
 language: java
+jdk:
+  - oraclejdk11
```

Comment:
Is the Oracle JDK more popular than OpenJDK?

Reply:
Primarily for consistency, as that's what we have on `tuna`:

```
jimmylin@tuna:~$ java -version
java version "11.0.4" 2019-07-16 LTS
Java(TM) SE Runtime Environment 18.9 (build 11.0.4+10-LTS)
Java HotSpot(TM) 64-Bit Server VM 18.9 (build 11.0.4+10-LTS, mixed mode)
```